cost-analyzer/templates/_helpers.tpl

```diff
@@ -992,7 +1006,7 @@ Begin Kubecost 2.0 templates
       readOnly: true
   {{- end }}
   {{- if (.Values.kubecostProductConfigs).cloudIntegrationSecret }}
-    - name: {{ .Values.kubecostProductConfigs.cloudIntegrationSecret }}
+    - name: cloud-integration
```

Comment:
Oh wait. Are we sure we want to be hardcoding this to `cloud-integration`? I think the primary requirement is that the key is named `cloud-integration.json`, but the secret can have any name?

Reply:
this looks like the name of the volume that he is mounting the secret as - is internal to the pod spec

Reply:
Ah! I see. It's the name of the volume:

https://github.com/kubecost/cost-analyzer-helm-chart/blob/456f992b65d2652b625a1996aaa3dedf9ca00e9b/cost-analyzer/templates/aggregator-cloud-cost-deployment.yaml#L54

Thanks for clarifying.

Reply:
Correct, there is no reason the name of the volume needs to be influenced by the value. What matters is the value be pointing to the right Secret.